SPECS/nodejs/nodejs.spec

```
@@ -6,8 +6,8 @@ Name:           nodejs
 # WARNINGS: MUST check and update the 'npm_version' macro for every version update of this package.
 #           The version of NPM can be found inside the sources under 'deps/npm/package.json'.
 Version:        16.18.1
-Release:        1%{?dist}
-License:        BSD and MIT and Public Domain and naist-2003
+Release:        2%{?dist}
+License:        BSD and MIT and Public Domain and naist-2003 and Artistic 2.0
```

Nit-pick: we've recently switched to trying to use [SPDX identifiers](https://spdx.org/licenses/) instead of Fedora's short names, although I don't know, if there's been any specific announcement regarding this. Anyway, for the sake of following that guidance:
```suggestion
License:        BSD and MIT and Public Domain and naist-2003 and Artistic-2.0
```